meerk40t/core/elements.py

```diff
@@ -5540,6 +5541,79 @@ def contains(box, x, y=None):
         self._emphasized_bounds = None
         self.set_emphasis(None)
 
+    def classify_legacy(self, elements, operations=None, add_op_function=None):
+        """
+        Classify does the placement of elements within operations.
+        "Image" is the default for images.
+        Typically,
+        If element strokes are red they get classed as cut operations
+        If they are otherwise they get classed as engrave.
+        However, this differs based on the ops in question.
+        :param elements: list of elements to classify.
+        :param operations: operations list to classify into.
+        :param add_op_function: function to add a new operation, because of a lack of classification options.
+        :return:
+        """
+        if elements is None:
+            return
+        reverse = self.context.classify_reverse
+        if reverse:
+            elements = reversed(elements)
+        if operations is None:
+            operations = list(self.ops())
+        if add_op_function is None:
+            add_op_function = self.add_op
+        for element in elements:
+            was_classified = False
+            if hasattr(element, "operation"):
+                add_op_function(element)
+                continue
+            if element is None:
+                continue
+            for op in operations:
+                if op.operation == "Raster":
+                    if element.stroke is not None and op.color == abs(element.stroke):
+                        op.add(element, type="opnode")
+                        was_classified = True
+                    elif isinstance(element, SVGImage):
+                        op.add(element, type="opnode")
+                        was_classified = True
+                    elif isinstance(element, SVGText):
```

Comment:
This should be left as-is for compatibility but it adds a text element with no fill and very thin stroke to a raster operation which is not really correct.

Reply:
I can't really do much else with them and it's a one text fits all thing. So all text is treated the same way regardless of other factors.

Reply:
I did say it needed to be left. Just pointing out a use case where this legacy approach is incorrect - as I keep saying there are lots of use cases and getting them all correct is impossible, and getting most of them correct in a consistent way is bloody difficult.